docs/reference/commandline/exec.md

```diff
@@ -39,6 +39,14 @@ The command started using `docker exec` only runs while the container's primary
 process (`PID 1`) is running, and it is not restarted if the container is
 restarted.
 
+> **Note**: COMMAND will run in the default directory of the container. It the
+> underlying image has a custom directory specified with the WORKDIR directive
+> in its Dockerfile, this will be used instead.
+
+> **Note**: COMMAND should be an executable, a chained or a quoted command
```

Comment:
So now that I see these together I am going to be picky and say no note-stacking. I think you don't actually need either of these to be notes. Just make them paragraphs. :)